src/app/core/entity-components/entity-utils/view-title/view-title.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, Input } from "@angular/core";
+import { getUrlWithoutParams } from "../../../../utils/utils";
+import { Router } from "@angular/router";
+
+@Component({
+  selector: "app-view-title",
+  templateUrl: "./view-title.component.html",
+  styleUrls: ["./view-title.component.scss"],
+})
+export class ViewTitleComponent {
+  /** The page title to be displayed */
+  @Input() title: string;
+
+  /** (Optional) do not show button to navigate back to the parent page */
+  @Input() disableBackButton: boolean = false;
+
+  readonly parentUrl: string;
+
+  constructor(private router: Router) {
+    this.parentUrl = this.findParentUrl();
+  }
+
+  private findParentUrl(): string {
+    const currentUrl = getUrlWithoutParams(this.router);
+    const lastUrlSegmentStart = currentUrl.lastIndexOf("/");
```

Comment:
In some cases this is not the expected behavior. E.g. accidentally clicking on a school in the education section of a child. When navigating back, you end up on the school overview instead of the child from which you came from.

Reply:
reverted to the basic "history back" navigation for now, component supports both hierarchical and history.back through a flag